src/services/user.js

```
@@ -100,7 +101,16 @@ const userService = {
 
   deleteUser: async (id) => {
     await User.findByIdAndRemove(id).exec()
-  }
+  },
+  setConfirmTokenToTrue : async (id) => {
+    console.log(id)
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)